Flatten a list.py

```diff
@@ -0,0 +1,15 @@
+def flat_list(list_with_nested_lists, flat_result=None):
+    if flat_result is None:         # It's because not allowed to change default arguments

```

Comment:
не совсем понял что это 
> It's because not allowed to change default arguments
значит, но да, в питоне просто так пустой список значением по умолчанию не сделаешь

Здесь можно обойтись и без этого параметра, попробуй решить не меняя сигнатуру функции